libraries/AP_HAL_ChibiOS/hwdef/LongBowF405WING/Readme.md

```diff
@@ -0,0 +1,100 @@
+# LongBowF405WING Flight Controller
+
+The LongBowF405WING is a flight controller produced by [lefei rc](http://www.lefeirc.com/).
+
```

Comment:
insert link to images here
also need an image of the PDB since it has external connections (probably ESCpwer out and VBAT in)